x-pack/test_serverless/functional/test_suites/security/ftr/cloud_security_posture/compliance_dashboard.ts

```diff
@@ -43,7 +43,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
     let dashboard: typeof pageObjects.cloudPostureDashboard.dashboard;
 
     before(async () => {
-      await pageObjects.svlCommonPage.loginWithRole('viewer');
+      await pageObjects.svlCommonPage.loginWithPrivilegedRole();
```

Comment:
missed during the initial review, but as I see similar questions regarding other tests - is there any reason to change here from the least privileged role `viewer` to the privileged one?

Reply:
https://github.com/elastic/kibana/pull/188128/commits/47c0c5d85eff8e722c502b3837aac80d53d8ac14